packages/dcos-integration-test/extra/test_3dt.py

```diff
@@ -283,6 +283,27 @@ def test_3dt_units(cluster):
         )
 
 
+@retrying.retry(wait_fixed=2000, stop_max_delay=LATENCY*1000)
+def test_systemd_units_health(cluster):
+    """
+    test all units and make sure the units are healthy. This test will fail if any of systemd unit is unhealthy,
+    meaning it focuses on making sure the cluster is healthy, rather then testing 3dt itself.
+    """
+    assert cluster.masters, "Must have at least 1 master node"
+    node_response = make_3dt_request(cluster.masters[0], BASE_ENDPOINT_3DT + '?cache=0', cluster, port=PORT_3DT)
+    for unit in node_response['units']:
+        assert 'id' in unit, "Unit response {} is missing `id` field".format(unit)
+        assert 'health' in unit, "Unit response {} does not have `health` field".format(unit)
+        if unit['health'] != 0:
+            # unhealthy unit found
+            report_response = make_3dt_request(cluster.masters[0], BASE_ENDPOINT_3DT + '/report', cluster,
+                                               port=PORT_3DT)
+            logging.info(report_response)
+            raise AssertionError('Unhealthy unit {} has been found, health status {}. Please see full dump report to '
```

Comment:
any chance we can dump the journalctl of that unit right here? (Doesn't the 3dt api have a bit of that somewhere?)